index.js

```diff
@@ -125,12 +125,34 @@ var torrentStream = function (link, opts, cb) {
     }
   })
 
+  var handleReserved = function (p) {
+    this.replaceAt = function (str, index, character) {
+      return str.substr(0, index) + character + str.substr(index + character.length)
+    }
+
+    var reser = ['<', '>', ':', '"', '/', '\\', '|', '?', '*']
+    var s = p.split(path.sep)
+    var root = path.parse(p).root
+    var n = []
+    for (var i = 0; i < s.length; ++i) {
+      var hold = s[i]
+      for (var cnt = 0; cnt < hold.length && (s[i] + path.sep) !== root; ++cnt) {
+        if (reser.indexOf(hold[cnt]) !== -1) {
+          hold = this.replaceAt(hold, cnt, '_')
```

Comment:
Yes , you are right, i just wanted the project owners to get into the point that its not only parsing the file name, but the full path of the file. Anyway, ive made a new modification using regexp and `.replace(),` i hope you have a look to it. 
thanks,